azure-samples/src/main/java/com/microsoft/azure/management/compute/samples/ManageVirtualMachine.java

```diff
@@ -152,8 +152,8 @@ public static void main(String[] args) {
                 //=============================================================
                 // Update - Expand the OS drive size by 10 GB
 
-                Integer osDiskSizeInGb = vm.osDiskSize();
-                if (osDiskSizeInGb == null) {
+                int osDiskSizeInGb = vm.osDiskSize();
+                if (osDiskSizeInGb == 0) {
```

Comment:
would this ever happen in a normal scenario?  I think someone mentioned that it would for auto-created disks?